python/tvm/driver/tvmc/common.py

```diff
@@ -415,3 +415,86 @@ def parse_shape_string(inputs_string):
         shape_dict[name] = shape
 
     return shape_dict
+
+
+def set_config_value(name, value, config_type):
```

Comment:
Just a semantic nit here: it seems it's more a "get" than a "set" function? Like `get_pass_config_value` (also taking into account the suggestion from @comaniac about using `pass-config` flag instead of `config`, which I liked :)

Reply:
Makes sense, I updated that now.